org.freedesktop.Sdk.Extension.scala.json

```diff
@@ -0,0 +1,54 @@
+{
+  "id": "org.freedesktop.Sdk.Extension.scala",
+  "branch": "25.08",
+  "runtime": "org.freedesktop.Sdk",
+  "runtime-version": "25.08",
+  "sdk": "org.freedesktop.Sdk",
+  "build-extension": true,
+  "separate-locales": false,
+  "modules": [
+    {
+      "name": "scala",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "archive",
+          "archive-type": "tar-gzip",
+          "url": "https://github.com/scala/scala3/releases/download/3.7.4/scala3-3.7.4-x86_64-pc-linux.tar.gz",
```

Comment:
why not compile from source to support both x86_64 and aarch64? For an SDK extension, restricting it to one arch limits adoption among apps using it.